app/views/content_items/manuals/_manual_layout.html.erb

```diff
@@ -17,17 +17,6 @@
 
     <%= yield %>
 
-    <%= render 'govuk_publishing_components/components/print_link', {
-      data_attributes: {
-        module: "ga4-event-tracker",
-        ga4_event: {
-          event_name: 'print_page',
-          type: 'print page',
-          index_link: 1,
-          index_total: 1,
-          section: 'Footer',
-        },
-      },
-    } %>
+    <%= render 'govuk_publishing_components/components/print_link', {} %>
```

Comment:
This doesn't need the curly braces, it can just be:

```
<%= render 'govuk_publishing_components/components/print_link' %>
```

(also in a few other places)

Reply:
Thanks @andysellick - good spot, will fix this then merge :+1: